events/class_tests/create_meeting_form.py

```diff
@@ -0,0 +1,195 @@
+                    'participants-0-participant_email': ''
+                },
+                "You have to enter at least one participant in the meeting"
+        ),
+
+    ], ids=[
+        "user does not exist",
+        "event creator email included in participant",
+        "empty participant",
+    ]
+                             )
+    def test_invalid_participant_formset(self, invalid_participant_data, expected_error):
+        user_id = User.objects.get(pk=1)
+        view = self.create_meeting_view(user_id, participant_data=invalid_participant_data)
+        assert not view.meeting_participants_formset.is_valid()
+        assert expected_error in view.meeting_participants_formset.non_form_errors()
+
+    @pytest.mark.parametrize("invalid_optional_meeting_data,expected_error,is_formset_error", [
+        # blank end date
+        (
+                {'optional_meetings-TOTAL_FORMS': '1', 'optional_meetings-INITIAL_FORMS': '0',
+                 'optional_meetings-0-date_time_start': OPTIONAL_MEETING_DATE_START,
+                 'optional_meetings-0-date_time_end': '',
+                 },
+                "'Ending date cannot be blank'", False
+        ),
+        (
+                # blank start date
+                {'optional_meetings-TOTAL_FORMS': '1', 'optional_meetings-INITIAL_FORMS': '0',
+                 'optional_meetings-0-date_time_start': '',
+                 'optional_meetings-0-date_time_end': OPTIONAL_MEETING_DATE_END,
+                 },
+                "Starting date cannot be blank", False
+        ),
+        (
+                # dates not unique
+                {'optional_meetings-TOTAL_FORMS': '2', 'optional_meetings-INITIAL_FORMS': '0',
+                 'optional_meetings-0-date_time_start': OPTIONAL_MEETING_DATE_START,
+                 'optional_meetings-0-date_time_end': OPTIONAL_MEETING_DATE_END,
+                 'optional_meetings-1-date_time_start': OPTIONAL_MEETING_DATE_START,
+                 'optional_meetings-1-date_time_end': OPTIONAL_MEETING_DATE_END,
+                 },
+                "The optional meeting dates should be different", True
+        ),
+        (
+                # dates in the past
+                {'optional_meetings-TOTAL_FORMS': '1', 'optional_meetings-INITIAL_FORMS': '0',
+                 'optional_meetings-0-date_time_start': OPTIONAL_MEETING_DATE_START_IN_THE_PAST,
+                 'optional_meetings-0-date_time_end': OPTIONAL_MEETING_DATE_END_IN_THE_PAST,
+                 },
+                "Optional meeting dates cannot be in the past", True
+        )
+    ], ids=[
+        "blank end date",
+        "blank start date",
+        "dates not unique",
+        "dates in the past",
+    ]
+                             )
+    def test_invalid_optional_meeting_formset(self, invalid_optional_meeting_data, expected_error, is_formset_error):
+        view = self.create_meeting_view(optional_meeting_data=invalid_optional_meeting_data)
```

Comment:
Same as above 😄 
In my opinion, you can use the client.post instead of creating a meeting view